README.md

```
@@ -1,50 +1,183 @@
+- [@imxade](https://github.com/imxade) (Rituraj)
+
+## 📍 License
+
+This project is licensed under the **Apache License 2.0**. See the [LICENSE](./LICENSE) file for details.
+
+## 💪 Thanks To All Contributors
+
+Thanks a lot for spending your time helping Rein grow. Keep rocking! 🥂
+
+<a href="https://github.com/AOSSIE-Org/rein/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=AOSSIE-Org/rein" />
```

markdownlint-cli2 (0.20.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)